### ISSUE TITLE
Adding deleted user

### ISSUE THREAD

**Issue by pintman** [friendica/friendica#8192]
### Expected behavior

Adding a deleted user as contact should not be possible.

### Actual behavior

There is an error when running on opensocial.at

```
Warning: count(): Parameter must be an array or an object that implements Countable in /var/www/html/mod/dfrn_request.php on line 65

Warning: count(): Parameter must be an array or an object that implements Countable in /var/www/html/mod/dfrn_request.php on line 66

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/mod/dfrn_request.php:65) in /var/www/html/src/App/Page.php on line 437

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/mod/dfrn_request.php:65) in /var/www/html/src/App/Page.php on line 438

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/mod/dfrn_request.php:65) in /var/www/html/src/App/Page.php on line 445

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/mod/dfrn_request.php:65) in /var/www/html/src/App/Page.php on line 446

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/mod/dfrn_request.php:65) in /var/www/html/src/App/Page.php on line 447

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/mod/dfrn_request.php:65) in /var/www/html/src/App/Page.php on line 448
```

### Steps to reproduce the problem

1. Create a user.
2. Delete the user.
3. Create another user.
4. Try to add the deleted user as contact.

### Friendica version you encountered the problem

2019.12

### Friendica source (git, zip)

I don't know.

### PHP version

I don't know.

### SQL version

I don't know.

**Comment by MrPetovan** [comment]
Are you actually able to add a deleted user profile as a contact? Not sure I understand the relationship with the error you provided.

**Comment by pintman** [comment]
Right. I was adding a deleted user as contact.

**Comment by MrPetovan** [comment]
And instead of an expected error in a regular page, you got a white page with the PHP error message?

**Comment by pintman** [comment]
Correct, I get a broken layout and the error message within.

**Comment by MrPetovan** [comment]
Thanks for the report, it looks like a regular Warning PHP message, next time can you please post it in #7999 to limit the number of open issues.

The reason why you have a broken layout is because your web server is configured to display PHP messages in the browser. This is not recommended, including because of breaking the layout for no good reason, and leaking of some server information (file paths, etc...).

In the end the actual behavior was as expected, you've not been able to connect to a deleted profile, and the worst inconvenience was the broken layout caused by unrecommended PHP settings (look into `display_errors` in your `php.ini`).

**Comment by pintman** [comment]
OK, thanks for clarifying this. I will report this to the instance admin.

**Comment by nupplaphil** [comment]
Hm .. I didn't enabled the `display_errors` explicit anywhere in my friendica environment. I think it's somwhere per default "on" if nothing is set (Maybe I open an issue in the docker-php github for it ..).

Nevertheless, I explicitly **disabled** it now. So the errors shouldn't get printed to the frontpage anymore :) Thanks for your investigation!

**Comment by MrPetovan** [comment]
Closed by #8204
